x-pack/plugins/ingest_manager/server/routes/package_policy/handlers.test.ts

```diff
@@ -168,45 +168,53 @@ describe('When calling package policy', () => {
         const request = getCreateKibanaRequest();
         await routeHandler(context, request, response);
         expect(response.ok).toHaveBeenCalled();
-        expect(callbackOne).toHaveBeenCalledWith({
-          policy_id: 'a5ca00c0-b30c-11ea-9732-1bb05811278c',
-          description: '',
-          enabled: true,
-          inputs: [],
-          name: 'endpoint-1',
-          namespace: 'default',
-          output_id: '',
-          package: {
-            name: 'endpoint',
-            title: 'Elastic Endpoint',
-            version: '0.5.0',
+        expect(callbackOne).toHaveBeenCalledWith(
+          {
+            policy_id: 'a5ca00c0-b30c-11ea-9732-1bb05811278c',
+            description: '',
+            enabled: true,
+            inputs: [],
+            name: 'endpoint-1',
+            namespace: 'default',
+            output_id: '',
+            package: {
+              name: 'endpoint',
+              title: 'Elastic Endpoint',
+              version: '0.5.0',
+            },
           },
-        });
-        expect(callbackTwo).toHaveBeenCalledWith({
-          policy_id: 'a5ca00c0-b30c-11ea-9732-1bb05811278c',
-          description: '',
-          enabled: true,
-          inputs: [
-            {
-              type: 'endpoint',
-              enabled: true,
-              streams: [],
-              config: {
-                one: {
-                  value: 'inserted by callbackOne',
+          expect.anything(), // request context
+          expect.anything() // kibanarequest
+        );
+        expect(callbackTwo).toHaveBeenCalledWith(
+          {
+            policy_id: 'a5ca00c0-b30c-11ea-9732-1bb05811278c',
+            description: '',
+            enabled: true,
+            inputs: [
+              {
+                type: 'endpoint',
+                enabled: true,
+                streams: [],
+                config: {
+                  one: {
+                    value: 'inserted by callbackOne',
+                  },
                 },
               },
+            ],
+            name: 'endpoint-1',
+            namespace: 'default',
+            output_id: '',
+            package: {
+              name: 'endpoint',
+              title: 'Elastic Endpoint',
+              version: '0.5.0',
             },
-          ],
-          name: 'endpoint-1',
-          namespace: 'default',
-          output_id: '',
-          package: {
-            name: 'endpoint',
-            title: 'Elastic Endpoint',
-            version: '0.5.0',
           },
-        });
+          expect.anything(), // request context
+          expect.anything() // kibanarequest
```

Comment:
if we want these to be a bit more explicit, just say so. not sure what level of specificity would be desired. or how to test for those types